adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/TokenCache.cs

```diff
@@ -172,6 +172,26 @@ public byte[] Serialize()
             }
         }
 
+        /// <summary>
+        /// Serializes current state of the cache as a blob. Caller application can persist the blob and update the state of the cache later by 
+        /// passing that blob back in constructor or by calling method Deserialize.
+        /// </summary>
+        /// <returns>
+        /// A tuple containing the following information:
+        /// <item><see cref="Tuple{T1,T2}.Item1"/>: Current state of the Adal cache as a blob</item>
+        /// <item><see cref="Tuple{T1,T2}.Item2"/>: Current state of the Unified cache as a blob</item>
+        /// </returns>
+        public Tuple<byte[], byte[]> SerializeAdalAndUnifiedCache()
```

Comment:
Why `Unified`? I think it should be `SerializeAdalAndMsalCache`

Reply:
disagree. This is no longer the MSAL cache, this is the new cache which works both for MSAL and ADAL and across platforms. Unified seems like an elegant name :)

---
In reply to: [211679733](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1217#discussion_r211679733) [](ancestors = 211679733)

Reply:
I see...makes sense.

Reply:
I second Henrik